buildfix.sh

```diff
@@ -37,7 +37,11 @@ echo "Formatting WORKSPACE/BUILD files..."
 buildifier -r .
 
 echo "Building and running gofmt..."
-bazel run "${BAZEL_QUIET_FLAGS[@]}" //:gofmt -- -w .
+gofmt_exit_code=$(bazel run "${BAZEL_QUIET_FLAGS[@]}" //:gofmt -- -w . || echo $?)
+
+if [ "$gofmt_exit_code" -ne 0 ]; then
+  echo "Error occurred during gofmt, but continuing..."
+fi
```

Comment:
If `bazel run` puts anything on stdout other than empty string `""` then this logic will break.

Can instead write

```
bazel run "${BAZEL_QUIET_FLAGS[@]}" //:gofmt -- -w . ||
    echo "Error occurred during gofmt, but continuing..."
```